contrib/ansible/openshift/register.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+default_tools_interval=$1
+ose_master_interval=$2
+ose_node_interval=$3
+file="/tmp/file"
+tool_dir=/var/lib/pbench-agent/tools-default
+
+pbench-stop-tools
+pbench-kill-tools
+pbench-clear-tools
+
+while read -u 9 line;do
+  remote=$(echo $line | awk -F' ' '{print $1}')
+  group=$(echo $line | awk -F' ' '{print $2}')
+  index=$(echo $line | awk -F' ' '{print $3}')
+  index=$(( index + 1 ))
+  label=svt_"$group"_"$index"
+## check if a remote is already registered
+  for tool in `/bin/ls $tool_dir`; do
+    remote_host=$(echo "$tool" | grep "^remote")
+    remote_hostname=$(echo "$remote_host" | awk -F@ '{print $2}')
+    if [ "$remote" == "$remote_hostname" ]; then
+      truncate -s -1 $tool_dir/$tool
+      printf _"$group"_"$index" >> $tool_dir/$tool
```

Comment:
And create a `$remote.tools` file which has the list of tools to register for that host.